src/scheduled.js

```diff
@@ -27,7 +27,13 @@ const readLobstersRss = (event) => {
         lastSeen = publishedAt;
         
         event.logger.info('Broadcasting story.', {itemDate: item.isoDate, itemGuid: item.guid, lastSeen});
-        event.reply(`${item.title} [${item.categories.join(' ')}] (${username}) ${item.guid}`);
+        
+        if(item.link === item.comments) {
+          event.reply(`${item.title} [${item.categories.join(' ')}] (${username}) ${item.guid}`);          
+        }
+        else {
+          event.reply(`${item.title} {${item.link}} [${item.categories.join(' ')}] (${username}) ${item.guid}`);
```

Comment:
Can we check the length of the message before we send it? I feel like adding the URL can create a message that's too long for IRC.

Reply:
Hi @hmadison ,

Is the check really necessary ? Recently I got a list of all URLs posted to lobsters from @pushcx  and I created a histogram of these which you can find below. Most of the URLs lengths are in range of 30 to 80 characters which seems to be fine IMHO. 

![Figure_1](https://user-images.githubusercontent.com/56976873/120625758-49dd2180-c462-11eb-81c2-d62e9f0b8a81.png)

Reply:
Pulling the maximum lengths from the schema.rb, we have an upper bound of 707 characters. IRC's max length is 510.

Reply:
Ah okay yes. In that case we need to add checks for length. I'll make the necessary changes. Please do make sure to run some tests on them since I am not familiar with javascript

Reply:
git.io should work if desired - https://github.blog/2011-11-10-git-io-github-url-shortener/

Reply:
@winks `git.io` is for GitHub controlled properties.